build-n-publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+pkgName="unlayer-editor"
```

Comment:
I think with this package name this url won't work: https://cdn.fromdoppler.com/unlayereditor/manifest/v1/asset-manifest.json (From [this PR](https://github.com/MakingSense/Doppler/pull/8411))

Reply:
Path correct: https://cdn.fromdoppler.com/unlayer-editor/manifest/v1/asset-manifest.json